SimPPS/PPSPixelDigiProducer/src/RPixLinearChargeDivider.cc

```diff
@@ -0,0 +1,114 @@
+#include "SimPPS/PPSPixelDigiProducer/interface/RPixLinearChargeDivider.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include "DataFormats/GeometryVector/interface/LocalVector.h"
+#include "Geometry/VeryForwardGeometry/interface/CTPPSPixelTopology.h"
+
+RPixLinearChargeDivider::RPixLinearChargeDivider(const edm::ParameterSet &params,  CLHEP::HepRandomEngine& eng,
+						 uint32_t det_id) : params_(params), rndEngine(eng) , _det_id(det_id)
+{
+  verbosity_ = params.getParameter<int>("RPixVerbosity");
+  fluctuate = new SiG4UniversalFluctuation();
```

Comment:
Can this be a regular member instead of pointer?

Reply:
it needs to be a pointer, otherwise a deeper change in the code is needed.

Reply:
How about a `std:unique_ptr` then?